src/zod/zod.test.ts

```diff
@@ -34,7 +44,27 @@ describe('AbiSchema', () => {
       ]
     `)
   })
-
+  it('successfully parses ensRegistryWithFallbackAbi', () => {
+    Abi.parse(ensRegistryWithFallbackAbi)
+  })
+  it('successfully parses nestedTupleArrayAbi', () => {
+    Abi.parse(nestedTupleArrayAbi)
+  })
+  it('successfully parses nounsAuctionHouseAbi', () => {
+    Abi.parse(nounsAuctionHouseAbi)
+  })
+  it('successfully parses wagmiMintExampleAbi', () => {
+    Abi.parse(wagmiMintExampleAbi)
+  })
+  it('successfully parses wethAbi', () => {
+    Abi.parse(wethAbi)
+  })
+  it('successfully parses writingEditionsFactoryAbi', () => {
+    Abi.parse(writingEditionsFactoryAbi)
+  })
+  it('successfully parses customSolidityErrorsAbi', () => {
+    Abi.parse(customSolidityErrorsAbi)
+  })
```

Comment:
let's group all these at the bottom in a `describe('ABI')` and chance the names to `it('nameOfAbi')`